docs/develop/accessibility/accessibility-testing.md

```diff
@@ -0,0 +1,109 @@
+# Testing the accessibility features
+
+The accessibility features are validated using a test process. This guide describes how to test and validate the UI elements of **Microsoft MakeCode** and targets.
+
+## Testing sites and targets
+
+Makecode.com, MakeCode targets, and documentation must be tested for these core features:
+
+* [Keyboard navigation](#testing-keyboard-navigation)
+* [Assistive technologies](#testing-assistive-technologies)
+* [High contrast](#testing-high-contrast)
+
+There are some exceptions to the accessibility test coverage. These external resources are excluded from testing:
+
+* MakeCode editor (PXT)/Help/Support
+* PXT/Help/Buy
+* PXT/Settings/Privacy & Cookies
+* PXT/Settings/Terms Of Use
+* PXT/Settings/Give Feedback
+* In MakeCode.com, any YouTube integration
+
+## Testing keyboard navigation
+
+Users must have the same access with the keyboard to UI elements that the mouse and touch input have in MakeCode.
+
+### Navigation path
+
+Using a left-to-right reading order, the user must be able to **navigate from left to right, from top to bottom** using the ``Tab`` key. The user must also be able to navigate in the opposite direction using the ``Shift + Tab`` keys.
+The simulator is exempt from this. Keyboard navigation in the simulator doesn't follow any reading order **yet**.
+
+![](/static/images/accessibility-keyboard-navigation.gif)
+
+### Tab panels
+
+By convention, tab panels must always be navigable with the ``Left``, ``Right``, ``Up`` and ``Down`` arrow keys. In right-to-left reading mode, the behavior must follow the reading order. The navigation must loop, which means that once the last tab position is reached, the next ``Tab`` key press takes the reader back to the first tab position.
+
+![](/static/images/accessibility-tabpanel.gif)
+
+### Dialog box
+
+Many dialog boxes are shown in PXT. Dialog boxes are used for the Projects button, Share, Language, About, and others. When a dialog box is open, the user must not have to access to any interactive element outside of the dialog prior to closing it. The keyboard navigation is 'trapped' inside the dialog.
+
+The dialog must close itself with activation of the answer buttons (OK, Accept, Cancel...), the close button, or by pressing ``Escape``.
+
+![](/static/images/accessibility-modals.gif)
```

Comment:
Can we move all accessibility gifs / images into their own folder under /static/images/accessibility/